packages/menu/src/Menu.tsx

```diff
@@ -340,17 +344,19 @@ function Menu({
       spacing={spacing}
       adjustOnMutation={adjustOnMutation}
     >
-      {/* Need to stop propagation, otherwise Menu will closed automatically when clicked */}
-      {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events*/}
-      <ul
-        {...rest}
-        className={cx(rootMenuStyle, className)}
-        role="menu"
-        ref={popoverRef}
-        onClick={e => e.stopPropagation()}
-      >
-        {updatedChildren}
-      </ul>
+      <div className={cx(rootMenuStyle, className)}>
+        {/* Need to stop propagation, otherwise Menu will closed automatically when clicked */}
```

Comment:
I bet this can be removed after https://github.com/mongodb/leafygreen-ui/pull/547 is merged

Reply:
(Referring to the onClick event handler below)

Reply:
We only changed the behavior of Tooltip, not Menu

Reply:
Ah Tooltip just copied this behavior

Reply:
We've discussed before but perhaps worth baking into Popover itself